김진/231205/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,7 @@
+function solution(s) {
+    let answer = [];
```

Comment:
```suggestion
    const answer = [];
```
![image](https://github.com/modern-agile-team/7term-assignment/assets/137868493/66324f47-0883-41b0-b3a5-c6d6f8bf8877)

https://nenara.com/75 참조여